src/server/game/Entities/Object/Object.h

```diff
@@ -906,7 +906,8 @@ class WorldObject : public Object, public WorldLocation
         float GetGridActivationRange() const;
         float GetVisibilityRange() const;
         float GetSightRange(const WorldObject* target = NULL) const;
-        bool CanSeeOrDetect(WorldObject const* obj, bool ignoreStealth = false, bool distanceCheck = false) const;
+        //bool CanSeeOrDetect(WorldObject const* obj, bool ignoreStealth = false, bool distanceCheck = false) const;
```

Comment:
Is there a reason to leave this as a comment? If it isn't needed anymore then remove it,

Reply:
Leave here for reference. In case of any bugs, we have a sample to compare to